pkg/storage/pebble.go

```diff
@@ -2063,6 +2063,13 @@ func (p *Pebble) GetTableMetrics(start, end roachpb.Key) ([]enginepb.SSTableMetr
 	return metricsInfo, nil
 }
 
+// GetAdvancedPebbleMetrics implements the Engine interface.
+func (p *Pebble) GetAdvancedPebbleMetrics(
+	start, end roachpb.Key,
+) ([]enginepb.AdvancedPebbleMetrics, error) {
+	return []enginepb.AdvancedPebbleMetrics{}, nil
```

Comment:
once https://github.com/cockroachdb/pebble/pull/2713 is merged, I will call ScanStatistics to retrieve the metrics.